src/parser/jobs/smn/Aetherflow.js

```diff
@@ -28,30 +28,45 @@ export default class Aetherflow extends Module {
 		'suggestions',
 	]
 
-	_badFesters = []
+	_badBanes = []			// In case we ever want to check for Banes where 1 or 0 DoTs are spread
+	_reallyBadBanes = [] 	// DoTless Bane...
+	_badFesters = []		// 1 DoT Fester. Oh no, it gets worse!
+	_reallyBadFesters = [] 	// 0 DoT Fester. :r333333:
 	_badPainflares = []
 
 	constructor(...args) {
 		super(...args)
+	   this.addHook('cast', {
+		   by: 'player',
+		   abilityId: ACTIONS.BANE.id,
+	   }, this._onCastSMNDOT)
 		this.addHook('cast', {
 			by: 'player',
 			abilityId: ACTIONS.FESTER.id,
-		}, this._onCastFester)
+	   }, this._onCastSMNDOT)
 		this.addHook('aoedamage', {
 			by: 'player',
 			abilityId: ACTIONS.PAINFLARE.id,
 		}, this._onPainflareDamage)
 		this.addHook('complete', this._onComplete)
 	}
 
-	_onCastFester(event) {
-		// Make sure all festers have the required dots up for Festers
+	_onCastSMNDOT(event) {
+		// Make sure all Festers & Banes have the required dots up
 		const target = this.enemies.getEntity(event.targetID)
-		const numStatuses = FESTER_STATUSES.filter(statusId => target.hasStatus(statusId)).length
-		if (numStatuses !== FESTER_STATUSES.length) {
-			this._badFesters.push(event)
-		}
-	}
+		const numStatuses = SMNDOT_STATUSES.filter(statusId => target.hasStatus(statusId)).length
+		
+		// Differentiates between partial (bad) or total (really bad) whiffs
+		if(numStatuses == 1){
+ 			this._badFesters.push(event)
+			this._badBanes.push(event)
```

Comment:
In both of these cases, we're pushing banes to the festers array and visa versa. Probably worth distinguishing between the two.